examples/brotli/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
```

Comment:
This should probably be replaced with 3.5.

Reply:
Yes please use 3.5 otherwise we get deprecation warnings while running this example

```suggestion
cmake_minimum_required(VERSION 3.5)
```

Reply:
fixed